PaulasWeatherApp/PaulasWeatherApp/ViewController/CurrentWeatherViewController.swift

```diff
@@ -8,11 +8,104 @@
 import UIKit
 
 class CurrentWeatherViewController: UIViewController {
+    
+    var tempLabel: UILabel = UILabel()
+    var sunriseLabel: UILabel = UILabel()
+    var sunsetLabel: UILabel = UILabel()
+    
+    let activityIndicator = UIActivityIndicatorView(style: .large)
+    
+    var sunriseString: NSMutableAttributedString = {
+        let iconsSize = CGRect(x: 0, y: -5, width: 15, height: 15)
+        let sunriseAttachment = NSTextAttachment()
+        sunriseAttachment.image = UIImage(systemName: "sunrise")
+        sunriseAttachment.bounds = iconsSize
+        let atr = NSMutableAttributedString(attachment: sunriseAttachment)
+        return atr
+    }()
+    
+    var sunsetString: NSMutableAttributedString = {
+        let iconsSize = CGRect(x: 0, y: -5, width: 15, height: 15)
+        let sunsetAttachment = NSTextAttachment()
+        sunsetAttachment.image = UIImage(systemName: "sunset")
+        sunsetAttachment.bounds = iconsSize
+        let atr = NSMutableAttributedString(attachment: sunsetAttachment)
+        return atr
```

Comment:
sunsetStrings and sunriseString have the exact same code for creation, except of the icon name. You could make a method that receives the image name and returns the created NSMutableAttributedString. In this was you avoid having the same code duplicated.